sql-parser/build.gradle

```diff
@@ -20,6 +20,7 @@ dependencies {
     compile "com.google.code.findbugs:jsr305:${versions.jsr305}"
     compile "com.google.guava:guava:${versions.guava}"
 
+    testImplementation project(':integration-testing')
```

Comment:
I'm not sure if it is worth pulling in the dependency to safe us from having to declare the `expectedException`. `:integration-testing` is very heavy and pulls in a lot.

I've had in mind to build some tooling that re-uses the sql-parser package standalone. I never got around to doing any of it, but I think it would be cool if we could keep this fairly isolated in terms of dependencies.